pypdf/_xobj_image_helpers.py

```diff
@@ -152,7 +152,7 @@ def _handle_flate(
     """
 
     def bits2byte(data: bytes, size: Tuple[int, int], bits: int) -> bytes:
-        mask = (2 << bits) - 1
+        mask = (1 << bits) - 1
```

Comment:
Could you please elaborate this change?

Reply:
This is a mis typing / bug inducing corruption in the image